sanctuarycomputer/contentful-fragment
# Draggable Reorder

Create grid and list-item wrapper components using the Muuri library to allow users to change the order of fragment cards by clicking and dragging.

**QA Instructions**
In application.js change line 5 to `const DEV = true;`
In your terminal `ember serve`.
Navigate to `localhost:4200/` to see the preview page.

Click and drag cards. Confirm the order is persisted by looking at the JSON preview.
Add a new card and/or edit data. Confirm data is persisted as expected.
